triton_backend/all_models/inflight_batcher_llm/preprocessing/1/model.py

```
@@ -136,8 +137,8 @@ def initialize(self, args):
                 'model_type']
 
             assert self.model_type in [
-                'llava', 'blip2-opt', 'vila', 'mllama', 'llava_onevision',
-                'qwen2_vl'
+                'llava', 'blip2-opt', 'pixtral', 'vila', 'mllama',
+                'llava_onevision', 'qwen2_vl'
             ], f"[TensorRT-LLM][ERROR] Currently supported multi-modal models are llava, blip2-opt, vila, mllama, llava_onevision and qwen2_vl. Got {self.model_type}."
```

Ruff (0.12.2)

Line too long (167 > 120)

(E501)